<launch>
 
  <!-- Global parameter available to all nodes -->
  <param name="global_param" value="global_value" />

  <!-- Launch the node with a private parameter -->
  <node pkg="parameter_test" type="param_first" name="param_first" output="screen">
    <!-- Private parameter for the node "my_node" -->
    <param name="private_param" value="node_specific_value" />
  </node>

</launch>
